code/game/machinery/pipe/construction.dm

```diff
@@ -256,8 +256,10 @@
 			return dir|flip
 		if(PIPE_SIMPLE_BENT, PIPE_INSULATED_BENT, PIPE_HE_BENT, PIPE_SUPPLY_BENT, PIPE_SCRUBBERS_BENT)
 			return dir //dir|acw
-		if(PIPE_CONNECTOR, PIPE_UVENT, PIPE_PASV_VENT, PIPE_SCRUBBER, PIPE_HEAT_EXCHANGE, PIPE_INJECTOR)
+		if(PIPE_CONNECTOR,  PIPE_HEAT_EXCHANGE, PIPE_INJECTOR)
 			return dir|flip
+		if(PIPE_UVENT, PIPE_PASV_VENT, PIPE_SCRUBBER)
+			return dir
```

Comment:
Can you not put these inside the `if` on line 257?

Reply:
Why not indeed, let me actually go ahead and do that

Reply:
Oh, it was to make clear the difference between 257 which was bent pipes, and how they also are applied the acw/cw, and vents and scrubbers which do not @SteelSlayer

Reply:
I'm really just referring to the return values. Both the new `if` you've added and the check on 257 does a `return dir`, therefore they should probably be combined.

Reply:
Probably won't make a difference when it comes to performance.